nextflow/nextflow.config

```diff
@@ -0,0 +1,105 @@
+manifest {
+  author = "Chunjie Liu"
+  homePage = "https://github.com/Xinglab/rmats-turbo"
+  mainScript = "rmatsTurbo.nf"
+}
+
+
+params {
+  // input bam files
+  bam_g1 = null
+  bam_g2 = null
+
+  // reference
+  gtf = null
+
+  is_single_end = false
+  readLength = null
+  nthread = 1
+
+  out_dir = null
+  lib_type = "fr-unstranded"
+  variable_read_length = false
+  anchorLength = false
+
+  tstat = 1
+  cstat = "0.0001"
+  statoff = false
+  paired_stats  = false
+  darts_model = false
+  darts_cutoff = "0.05"
+
+  novelSS = false
+  mil = 50
+  mel = 500
+  allow_clipping = false
+  individual_counts = false
+
+  machine_mem_gb = 4
+
+  rmats_version = "v4.3.0"
+
+
+  // Nextlfow
+  max_retries = 3
+  publishDir = ""
+
+  error_strategy = "finish"
+  allowed_error_strategies = ["terminate", "finish", "ignore", "retry"]
+  cleanup = false
+  echo = false
+  debug = false
+
+  mega_time = 20.h
+  tera_time = 50.h
+
+}
+
+cleanup = params.cleanup
+
+dsl2 {
+  enabled = true
+  endpoint = "-"
+}
+
+process {
+  errorStrategy = params.error_strategy
+  echo = params.echo
+
+  container = "xinglab/rmats:${params.rmats_version}"
+  executor = "slurm"
+
+  maxRetries = params.max_retries
+  maxErrors = '-1'
+
+  // Process-specific resource requirements
+  withLabel: mid_memory {
+    cpus = params.nthread
```

Comment:
This could be `cpus = 1` since `rmats_prep` uses `--nthread 1`

Reply:
it does not matter. In the rmats_prep, I set the --nthread 1.

The WDL used the same cpus and mem for the rmats_prep and rmats_post. Now I make it the mid_memory and tera_memory, users can change it by themselves.